### ISSUE TITLE
Feature / Song history

### ISSUE THREAD

**Issue by Bratanov** [Bratanov/community-driven-radio#5]
- Disallow songs played recently/duplicate songs in queue maybe?

**Comment by IvayloAtanasov** [comment]
I don't think we can have duplicates in queue now, maybe just the same song uploaded by a different guy (e.g. different youtube id)

For preventing recently played we can just have in memory queue of last played when the server is on. Next step could be writing them to db collection when/if we have db.
